estimate-web/src/utils/date.js

```diff
@@ -0,0 +1,28 @@
+const timeAgo = date => {
+    const seconds = Math.floor((new Date() - date) / 1000);
+
+    let interval = Math.floor(seconds / 31536000);
```

Comment:
Would make sense to put the numbers into constants so it's clearer what time unit do they represent